app/assets/stylesheets/layouts/_default.scss

```diff
@@ -0,0 +1,9 @@
+.layout-default {
```

Comment:
Based on our css structure, the layout style filename should be without `_` prefix
![Screen Shot 2021-06-18 at 6 15 24 PM](https://user-images.githubusercontent.com/6076762/122552977-3389b500-d061-11eb-88cb-e9cfd9526467.png)

Reply:
Woups!
Fixed :) 
Thanks!